pkg/thing/mocks/logger.go

```diff
@@ -0,0 +1,15 @@
+package mocks
+
+type FakeLogger struct{}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type FakeLogger should have comment or be unexported](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4853780)